lib/identity_config.rb

```diff
@@ -176,7 +176,7 @@ def self.build_store(config_map)
     config.add(:idv_public_key, type: :string)
     config.add(:idv_send_link_attempt_window_in_minutes, type: :integer)
     config.add(:idv_send_link_max_attempts, type: :integer)
-    config.add(:in_person_proofing_enabled, type: :boolean)
+    config.add(:in_person_proofing_enabled_issuers, type: :json, options: { symbolize_names: true })
```

Comment:
this is an array of strings, right? so no need to symbolize names?
```suggestion
    config.add(:in_person_proofing_enabled_issuers, type: :json)
```

Reply:
Ah yeah, good call.